README.md

```diff
@@ -79,40 +101,40 @@ It's safe to disable if you don't want those features.
 
 Graphical applications will run slowly without this.
 
-## allow=bluetooth
+### allow=bluetooth
 
 Only necessary when using "Real Wii remotes" in conjunction with the feature labeled "Emulate the Wii's Bluetooth adapter".
 
 It's safe to disable if that feature is not in use, generic bluetooth gamepads will still work without it. Actual Wii remotes can also work without this option when using the separate passthrough feature.
 
-## filesystem=xdg-run/app/com.discordapp.Discord:create
+### filesystem=xdg-run/app/com.discordapp.Discord:create
 
 Necessary for discord (a nonfree messaging service) integration.
 
 Can be safely dropped if discord is not used.
 
-## filesystem=xdg-run/gamescope-0:ro
+### filesystem=xdg-run/gamescope-0:ro
 
 Necessary for HDR10 support through gamescope.
 
 At the moment the only realistic usage case for this are HDR10 filters if you have an oled steam deck, it can safely be dropped in most other cases.
 
-## talk-name=org.freedesktop.ScreenSaver
+### talk-name=org.freedesktop.ScreenSaver
 
 Required for screensaver inhibition during gameplay.
 
 It can be disabled but your screensaver might trigger during gameplay depending on your input device and screensaver configuration.
 
-# dolphin-tool
+## dolphin-tool
 
-Some cli iso manipulation tasks can be achieved with `dolphin-tool`, it is bundled with the flatpak but not exposed to the outside.
+Some cli iso manipulation tasks can be achieved with `dolphin-tool`, it is bundled with the flatpak but not exposed through the Dolphin GUI.
```

Comment:
This makes it sound a bit like the tasks aren't doable through the dolphin gui even though they are.

What the original sentence was trying to say is that there's no entry point into the sandbox set up for the dolphin-tool binary, even though it is included.

Sorry to nitpick, I agree that the original wording is vague.

But maybe you can rewrite it a bit to make it clearer? Either that or leave it as it originally was, it's just a small detail.